src/setfit/modeling.py

```diff
@@ -21,7 +21,7 @@
 from sklearn.multiclass import OneVsRestClassifier
 from sklearn.multioutput import ClassifierChain, MultiOutputClassifier
 from torch.utils.data import DataLoader
-from tqdm.auto import tqdm
+from tqdm import trange
```

Comment:
```suggestion
from tqdm.auto import trange
```
The `tqdm.auto` import has support for progress bars in notebooks, whereas simply `from tqdm import ...` does not. See the docs [here](https://tqdm.github.io/docs/shortcuts/). (Although I suppose `tqdm.autonotebook` would also work as we don't use asyncio)